INSTALL.md

```diff
@@ -0,0 +1,84 @@
+# Installing device plugins to cluster
+
+## Install device plugins via a DaemonSet
+
+Each plugin can be installed via a DaemonSet. The install changes slightly based on the desired plugin. See install instructions per [plugin](README.md#plugins).
+
+Installing plugins via DaemonSets deployes them to the ```default``` (or currently active) namespace. Use kubectl's ```--namespace``` argument to change the deployment namespace.
+
+## Install device plugins via device plugin operator
+
+A more advanced install method is via device plugin operator. Operator configures plugin deployments based on the supplied device plugin CRDs (Custom Resource Definitions). See installation instructions in the [operator README](cmd/operator/README.md#installation).
+
+Operator installs device plugins to the ```inteldeviceplugins-system``` namespace.
+
+## Install with HELM charts
+
+Device plugins can also be installed to a cluster using the device plugin [operator Helm chart](https://github.com/intel/helm-charts/tree/main/charts/device-plugin-operator) (depending on cert-manager and NFD). Individual plugin projects are under https://github.com/intel/helm-charts/tree/main/charts/.
+
+As with the method above, these steps will install device plugin operator and plugins under ```inteldeviceplugins-system``` namespace.
+
+### Installing HELM repositories
+
+```bash
+helm repo add jetstack https://charts.jetstack.io # for cert-manager
+helm repo add nfd https://kubernetes-sigs.github.io/node-feature-discovery/charts # for NFD
+helm repo add intel https://intel.github.io/helm-charts/ # for device-plugin-operator and plugins
+helm repo update
+```
+
+### Installing cert-manager
+
+```bash
+helm install --wait \
+  cert-manager jetstack/cert-manager \
+  --namespace cert-manager \
+  --create-namespace \
+  --version v1.11.0 \
+  --set installCRDs=true
+```
+
+NOTE: cert-manager install takes a while to complete.
+
+### Installing NFD
+
+```bash
+helm install nfd nfd/node-feature-discovery \
+  --namespace node-feature-discovery --create-namespace --version 0.12.1 \
+  --set 'master.extraLabelNs={gpu.intel.com,sgx.intel.com}' \
+  --set 'master.resourceLabels={gpu.intel.com/millicores,gpu.intel.com/memory.max,gpu.intel.com/tiles,sgx.intel.com/epc}'
+```
+
+### Installing operator
```

Comment:
We will need a subsection that explains how to list available versions (with `--devel` helm param).

Reply:
Added something, can you check if it's in the right direction?

Reply:
looks OK, thanks!